src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php

```diff
@@ -88,11 +93,11 @@ protected function configure()
 message.
 
 Example running against a Bundle (AcmeBundle)
-  <info>php %command.full_name% --dump-messages en AcmeBundle</info>
+  <info>php %command.full_name% --dump-messages --sort=asc en AcmeBundle</info>
```

Comment:
I would remove the `--sort` in the 2 existing commands and add a sentence explaining the `sort` option with a new example.